src/components/full-calender/components/index.js

```diff
@@ -8,12 +8,16 @@ import Header from './shared/header';
 import ModelWindow from './shared/ModelWindow';
 import PriorityCheackBox from './shared/priority-checkBox';
 import React from 'react';
+import {convert} from '../components/shared/utilities';
 import dayGridPlugin from '@fullcalendar/daygrid';
 import interactionPlugin from '@fullcalendar/interaction';
 
 export default class EventCalender extends Component {
     date;
 
+    state = {
+        myDragEvent: ''
+    }
     handleEvents = (e) => {
         const eTarget= e.target;
```

Comment:
update touse descrutioring and renaming of the constants .

const {value:priorityId, checked:isSelected} = e.target